plenum/server/request_managers/request_manager.py

```diff
@@ -12,3 +13,23 @@ def static_validation(self, request: Request):
     @abstractmethod
     def dynamic_validation(self, request: Request):
         pass
+
+
+class RequestManager(AbstractRequestManager):
+
+    def __init__(self):
+        self.txn_types = set()
+        self.ledger_ids = set()
+        self.request_handlers = {}
+
+    def remove_req_handler(self, txn_type):
+        del self.request_handlers[txn_type]
+        self.txn_types.remove(txn_type)
+
+    def _register_req_handler(self, handler: RequestHandler):
```

Comment:
Do we really need this method? It looks like only `ReadRequestManager` uses it without modifications.

Reply:
and ActionRequestManager too